opentelemetry-sdk/CHANGELOG.md

```diff
@@ -52,6 +55,12 @@
   when its `shutdown` is invoked.
 
 - Reduced some info level logs to debug
+- **Breaking** for custom LogProcessor/Exporter authors:
+  Added an additional `name: Option<&str>` parameter to the `event_enabled`
```

Comment:
This isn't an addition of a new parameter. It's a change in the type of the existing parameter.

Reply:
oh yes that is true. I'll rephrase the wording.